modules/monitoring-persistent-storage-prerequisites.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-prerequisites_{context}"]
+== Persistent storage prerequisites
+
+ifdef::openshift-dedicated,openshift-rosa[]
+* Use the block type of storage.
+endif::openshift-dedicated,openshift-rosa[]
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* Dedicate sufficient persistent storage to ensure that the disk does not become full.
```

Comment:
Is there a guide or link to information on how to perform this action? Or how to verify that you have sufficient storage space?

Also - not sure why these prerequisites are a full module, rather than part of the configuring persistent storage module. Can you help me understand?

Reply:
For the first point, we have a link in additional resources that explains storage requirements. 
For the second point, I do not actually remember why I thought that was a good idea :smiley:  The only thing I can think of is that I did not like the idea of having two anchor IDs in one module, but that is not prohibited AFAIK, so I can put it back in the configuring persistent storage module.